Azure/sonic-buildimage
# [gbsyncd]: add gbsyncd to FEATURE table

**- Why I did it**
remove syncd from critical process list because
gbsyncd process will exit for platform without
gearbox.

closes #5623 

Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- How I did it**
remove syncd from critical process list because
gbsyncd process will exit for platform without
gearbox.

**- How to verify it**
apply the fix, reboot the box and check all gbsyncd related syslog message.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
